react/src/pages/SessionLauncherPage.tsx

```diff
@@ -1121,6 +1121,7 @@ const SessionLauncherPage = () => {
                       }
                       extraButtonTitle={t('button.Edit')}
                       onClickExtraButton={() => {
+                        setValidationTourOpen(false);
```

Comment:
Instead of setting `setValidationTourOpen` to `false`, you can put `SessionLauncherValidationTour` into the [final step]([url](https://github.com/lablup/backend.ai-webui/blob/main/react/src/pages/SessionLauncherPage.tsx#L1110-L1111)).